### PR TITLE
{tools} [goolf/1.7.20] IPython 4.2

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/i/IPython/IPython-4.2.0-goolf-1.7.20-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipykernel/'],
+    }),
+    ('pexpect', '4.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pexpect/'],
+    }),
+    ('backports.shutil_get_terminal_size', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports.shutil_get_terminal_size/'],
+    }),
+    ('pathlib2', '2.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pathlib2/'],
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-4.2.0-goolf-1.7.20-Python-2.7.11.eb]: listed twice?

File: easybuild/easyconfigs/i/IPython/IPython-4.2.0-goolf-1.7.20-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    ('certifi', '2016.2.28', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/certifi/'],
+    }),
+    ('backports_abc', '0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports_abc/'],
+    }),
+    ('tornado', '4.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado/'],
+    }),
+    ('Jinja2', '2.8', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/IPython/IPython-4.2.0-goolf-1.7.20-Python-2.7.11.eb]: apparently this requires `MarkupSafe`, cfr. #3087

**Comment by pescobar** [comment]
closing to retrigger tests

File: easybuild/easyconfigs/t/testpath/testpath-0.3-goolf-1.7.20-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = ['https://pypi.python.org/packages/py2.py3/t/testpath/']
+sources = ['testpath-%(version)s-py2.py3-none-any.whl']
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
+#builddependencies = [
+#    ('pip', '8.1.2', versionsuffix),
+#]
```
Comment (boegel) [inline easybuild/easyconfigs/t/testpath/testpath-0.3-goolf-1.7.20-Python-2.7.11.eb]: @pescobar what's this about?

Comment (boegel) [inline easybuild/easyconfigs/t/testpath/testpath-0.3-goolf-1.7.20-Python-2.7.11.eb]: ah, wait, we have `pip` as a part of the `Python` already, right? so just drop this

**Comment by boegel** [comment]
@pescobar sorry, seems like there was a good reason after all to list `pip` as a build dep for `testpath`:

```
Need pip version 8.0 or newer, found version 7.1.2')
```

not sure why you had it commented out though...

**Comment by pescobar** [comment]
@boegel I have pip 8 in my python easyconfig

**Comment by boegel** [comment]
@pescobar you'll need to add the easyconfig for pip itself too:

```
EasyBuildError: 'Irresolvable dependencies encountered: pip/8.1.2-goolf-1.7.20-Python-2.7.11'
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 8 (8 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/968bc7798be9ae1acbfa7a6a718768e2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/021c4aa0f46ef801a2e9aa7a73e224b9 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**